### ISSUE TITLE
Translate components in _app.js

### ISSUE THREAD

**Issue by Axiol** [vinissimus/next-translate#251]
I'm aware `_app.js` itself it not translatable out of the box. So, with this config

```
{
  "allLanguages": ["en", "fr"],
  "defaultLanguage": "fr",
  "defaultLangRedirect": "lang-path",
  "currentPagesDir": "pages_",
  "finalPagesDir": "pages",
  "localesPath": "locales",
  "pages": {
    "*": ["common"]
  }
}
```

I tried to as in the example. In my case, I have a `_app.js` I use as a template for my pages. So it contains a header component and a footer one. What I did is, inside the header component, is  to do as in the sample. So something like that

```
import Trans from 'next-translate/Trans';
import DynamicNamespaces from 'next-translate/DynamicNamespaces';
...
<DynamicNamespaces
        dynamic={(lang, ns) =>
            import(`../../locales/${lang}/${ns}.json`).then((m) => m.default)
        }
        namespaces={['dynamic']}
        fallback="Loading..."
        >
<Trans i18nKey="dynamic:myAccount" />
</DynamicNamespaces>
```

Compilation works fine. But when I load the page, I always end up with this error

```
Error: Cannot find module './/dynamic.json'
```

My guess is that it can't manage to get the current language. However I don't see how I can manage to get it properly

**Comment by aralroca** [comment]
@Axiol The error looks that the dynamic import is incorrect, can you verify that your `locales` is 2 folders up from the `_app.js`?

**Comment by cbfn** [comment]
Same problem here. Any solution for this particular case?

**Comment by aralroca** [comment]
Already fixed on [1.0.0-experimental.18](https://github.com/vinissimus/next-translate/releases/tag/1.0.0-experimental.18)

**Comment by aralroca** [comment]
Closing this. Already fixed on [1.0.0-canary.1](https://github.com/vinissimus/next-translate/releases/tag/1.0.0-canary.1). The 1.0 will be released in approximately 1-2 weeks. 😊
